tests/conftest.py

```diff
@@ -6,11 +6,17 @@
 import time
 from base64 import b64encode
 from sys import platform
+import warnings
 
 import betamax
 import pytest
 from betamax_serializers import pretty_json
 
+warnings.filterwarnings("error", category=DeprecationWarning)
```

Comment:
I don't think we need to add this as such warnings should not be emitted.

Reply:
I added that in as a check to make sure tests did not run, but it seems it is irrelevant since everything is imported from praw.models, which was renamed accordingly

Reply:
Oh does that make DeprecationWarnings errors?

Reply:
> Oh does that make DeprecationWarnings errors?

Yes

Reply:
I had to remove this code as it caused my coverage file to throw an exception